custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -63,13 +64,14 @@ def _get_credentials(self, repeat_record):
 
     def _base_properties(self, repeat_record):
         username, password = self._get_credentials(repeat_record)
+        server_ip = socket.gethostbyname(socket.gethostname())
```

Comment:
The [docs](https://docs.python.org/2.7/library/socket.html#socket.gethostname) for `socket.gethostname()` mention that "This operation assumes that there is a valid address-to-host mapping for the host, and the assumption does not always hold." Do you know what happens if that assumption does not hold? This might raise an exception, and we might want to catch it, because we might not want to prevent that from forwarding the record (although maybe we do, I don't know how critical the IP info is).